scene/2d/navigation_2d.cpp

```diff
@@ -29,675 +29,64 @@
 /*************************************************************************/
 
 #include "navigation_2d.h"
+#include "servers/navigation_2d_server.h"
```

Comment:
Nitpick but I like keeping a line of separation between the .cpp's own header and the rest:
```suggestion
#include "navigation_2d.h"

#include "servers/navigation_2d_server.h"
```